src/app/components/media/BulkActionsRemoveTag.js

```diff
@@ -0,0 +1,176 @@
+/* eslint-disable @calm/react-intl/missing-attribute */
```

Comment:
Please remove eslint-disable directive

Reply:
@alexmeedan removing this line shows `don't use spread operator  @calm/react-intl/missing-attribute` error